im_user.profile.inc

```diff
@@ -70,32 +64,27 @@ function _is_carrefour_market($profile) {
  */
 function _get_user_store($profile) {
   if(!isset($profile->grp)) return FALSE;
-  foreach ($profile->grp as $grp){
+  foreach ($profile->grp as $grp) {
     $counter = 0;
     $level   = '';
     $cn      = ''; 
     foreach($grp->attr as $attr){
-  	  if((string)$grp->attr[$counter]['type'] == 'level' && (string)$grp->attr[$counter] == 'AL') {
+  	  if ((string)$grp->attr[$counter]['type'] == 'level' && (string)$grp->attr[$counter] == 'AL') {
   		$level = 'AL';
   	  }
-      if((string)$grp->attr[$counter]['name'] == 'cn') {
+      if ((string)$grp->attr[$counter]['name'] == 'cn') {
   		$cn = (string)$grp->attr[$counter];
   	  }
-  	  if($level == 'AL' && $cn) {
+  	  if ($level == 'AL' && $cn) {
   	  	//assign if DATABASE table is present.
-	  	$result = db_query("SELECT pve_code
-		                        FROM {store_item_fields} sif 
-		                        WHERE sif.dit_cod_item = 'thales'
-		                        AND sif.ite_lib_value = :lib_value",
-		                        array(':lib_value' => $cn)
-		                       );
+	    $result = db_query("SELECT pve_code FROM {store_item_fields} sif WHERE sif.dit_cod_item = 'thales' AND sif.ite_lib_value = :lib_value", array(':lib_value' => $cn));

```

Comment:
this change has made the code less readable - it would be better to have line breaks in the string to allow for formatting of the SQL: begin the string with a linebreak, then the indentations will make more sense, and be much more readable - now you have a line that is probably too long and PHPCS is complaining about that.... you *are* using PHPCS, right?